src/layouts/WrapLayout.vue

```diff
@@ -1,30 +1,23 @@
 <template>
-    <div :style="style" class="container">
+    <div class="nvw-wrap-layout" :style="{flexDirection: orientation == 'horizontal' ? 'row' : 'column'}">
         <slot></slot>
     </div>
 </template>
 
 <script>
 export default {
   name: 'WrapLayout',
-  props: [
-    'orientation',
-    'itemWidth', // These props called itemWidth and itemHeight have to be reviewed in the meeting.
-    'itemHeight',
-  ],
-  data() {
-    return {
-      style: '',
-    };
-  },
-  mounted() {
-    this.style = `flex-direction: ${this.orientation == 'horizontal' ? 'row' : 'column'};`;
+  props: {
+    orientation: String,
+    // itemWidth and itemHeight will be set using directives for now.
+    itemWidth: Number,
```

Comment:
on nativescript-vue docs it says itemWidth and itemHeight but the examples use width and height. let's drop these from this component and use the common directive to process width and heoght for each children

Reply:
remove itemWidth and itemHeight props